openquake/hazardlib/gsim/bindi_2014.py

```diff
@@ -114,6 +114,27 @@ def _get_style_of_faulting_term(C, ctx):
     return C["sofN"] * NS + C["sofR"] * RS + C["sofS"] * SS
 
 
+def _get_dist_type(gmpe):
+    """
+    Get distance type required for the corresponding class of 
+    Bindi et al. (2014)
+    """
+    # Get gsim class name
+    gsim = gmpe.__class__().__class__.__name__
+    
+    rhypo_variants = [BindiEtAl2014Rhyp.__name__,
+                      BindiEtAl2014RhypEC8.__name__,
+                      BindiEtAl2014RhypEC8NoSOF.__name__]
+    
+    # Assign required distance type
+    if gsim in rhypo_variants:
+        dist_type = 'rhypo'
+    else:
+        dist_type = 'rjb'
+    
+    return [dist_type]
+
```

Comment:
Again, it is not clear why you are returning a list with a single argument and not a scalar.